src/Console/Command/DescribeCommand.php

```diff
@@ -359,7 +362,7 @@ private function getFixers(): array
 
         $fixers = [];
 
-        foreach ($this->fixerFactory->getFixers() as $fixer) {
+        foreach ($this->configResolver->getFixers() as $fixer) {
```

Comment:
this will limit the rules you can describe to the ones you configured I think?

Reply:
potentially we should get union here, built-in rules and registered custom ones [configured to be used, or not]